### PR TITLE
[FIX] portal: add loading on sign button

### PR DESCRIPTION
**Pull request by livaios**
opw-3709064

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by Demesmaeker — APPROVED** [review]
Hello :duck: 

Oh, nice!

Just tested previous version, and the problem wasn't present there, so... @robodoo r+ ^^

**Comment by Demesmaeker** [comment]
@robodoo r- Sorry, sorry, will let the other team see it too, my bad

**Review by chevalierv — COMMENTED** [review]
Hello @livaios

Good job on this task! Could you add some information about the fix in your commit message?

@Demesmaeker I hope you don't mind me hopping in. I couldn't resist the call of Javascript :stuck_out_tongue: 

Have a nice day!

File: addons/portal/static/src/signature_form/signature_form.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /** @odoo-module **/
 
 import { Component, onMounted, useRef, useState } from "@odoo/owl";
+import dom from "@web/legacy/js/core/dom";
```
[the remark this answers is not in the document]
Comment (livaios) [reply addons/portal/static/src/signature_form/signature_form.js:4]: Nice! That's good to know

**Review by chevalierv — APPROVED** [review]

**Comment by chevalierv** [comment]
@robodoo r+

**Pull request closed without merging**